### PR TITLE
feat(API-1895): create 'how to get app token' page

### PR DESCRIPTION
**Pull request by samokiss**
https://akeneo.atlassian.net/browse/API-1895

### CODE REVIEW

File: content/apps/create-app/callback-php.md [hunk 1]
```diff
@@ -24,25 +27,28 @@ if (empty($pimUrl)) {
...
 ];
 
-// Do a POST request on the token endpoint
-$ch = curl_init();
-curl_setopt($ch, CURLOPT_URL, $accessTokenUrl);
-curl_setopt($ch, CURLOPT_POST, true);
-curl_setopt($ch, CURLOPT_POSTFIELDS, $accessTokenRequestPayload);
-curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-$response = json_decode(curl_exec($ch), true);
+// If you haven't set your client yet, please install Guzzle by following the <a href="https://docs.guzzlephp.org/en/stable/overview.html#installation" target="_blank" rel="noopener noreferrer">official documentation</a>
```
Comment (tomglvng) [inline content/apps/create-app/callback-php.md:43]:
```suggestion
// If you haven't set your client yet, please install Guzzle by following the official documentation: https://docs.guzzlephp.org/en/stable/overview.html#installation
```
<a> is not interpreted in code markdown :(

File: content/apps/apps-how-to-get-your-app-token.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+{
+  "access_token": "Y2YyYjM1ZjMyMmZlZmE5Yzg0OTNiYjRjZTJjNjk0ZTUxYTE0NWI5Zm",
+  "token_type": "bearer",
+  "scope": "read_products write_products"
+}
+```
+
+You can use this token to call the Akeneo PIM REST API.
+
+## Next steps
```
Comment (tomglvng) [inline content/apps/apps-how-to-get-your-app-token.md:156]:
```suggestion

```

File: content/apps/apps-how-to-get-your-app-token.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+<div class="block-next-steps">
+    <img src="../img/illustrations/illus--Attribute.svg" width="140px">
+    <div class="block-next-steps-column">
+        <div class="block-next-steps-title">Next Step</div>
```
Comment (tomglvng) [inline content/apps/apps-how-to-get-your-app-token.md:161]:
```suggestion
        <div class="block-next-steps-title">Next Steps</div>
```

File: content/apps/apps-how-to-get-your-app-token.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+To know more about the step-by-step activation process, please read our article:  
+[How to connect an App?](https://help.akeneo.com/pim/serenity/articles/how-to-connect-my-pim-with-apps.html#how-to-connect-an-app)
+:::
+
+## Step 5: Use your access token to call the API
+
+At the end of this process, you receive the following response with an `access_token`:
+
+```json
+{
```
Comment (LevFlavien) [inline content/apps/apps-how-to-get-your-app-token.md:147]:
```suggestion

{
```

Comment (LevFlavien) [reply content/apps/apps-how-to-get-your-app-token.md:147]: (add line for snippet formatting)

File: src/index.handlebars [hunk 5]
```diff
@@ -9,7 +9,7 @@
                             <div class="hidden-xs hidden-sm col-md-6 col-lg-5">
                                 <h1 class="emphasis">Build Apps to ease the connection with Akeneo PIM</h1>
                                 <p>Follow our step-by-step guide to create an app and use OAuth 2.0</p>
-                                <a class="btn btn-primary" href="/apps/apps-getting-started.html">Build an App</a>
+                                <a class="btn btn-primary" href="/apps/apps-how-to-get-your-app-token.html">How to get App token</a>
```
Comment (tomglvng) [inline src/index.handlebars:12]:
```suggestion
                                <a class="btn btn-primary" href="/apps/overview.html">Build an App</a>
```
Let's not change the big button label, but redirect to App section

File: content/apps/apps-how-to-get-your-app-token.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+6. Your test App appears on the App Store page
+
+
+## Step 4: Connect your test App and access its settings
+
+![Test app on the App Store](../img/apps/marketplace-with-test-app.png)
+
+Connecting a test App is like connecting a published App. 
+
+1. Click on `Connect`
```
Comment (tomglvng) [inline content/apps/apps-how-to-get-your-app-token.md:124]: Add a first step "Launch your App"

File: content/apps/create-app/callback-php.md (hunk 1 above)
Comment (LevFlavien) [reply content/apps/create-app/callback-php.md:43]: may require to jump a line for visibility

```php
// If you haven't set your client yet, please install Guzzle by following the official documentation:
// https://docs.guzzlephp.org/en/stable/overview.html#installation
```

**Review by LevFlavien — APPROVED** [review]

**Pull request merged**